app/models/concerns/profile_tailoring.rb

```diff
@@ -12,6 +12,15 @@ def tailored_rule_ref_ids
     end).to_h
   end
 
+  def rule_group_ancestor_ref_ids
+    parent_groups = RuleGroup.where(id: added_rules.map(&:rule_group_id))
```

Comment:
I was thinking about `pluck` here, but the `added_rules` isn't implemented that way :disappointed: maybe something we can refactor. I'll look into it, until then this `map` is fine :+1: even though the query will look differently (see below).